src/plone/restapi/services/groups/update.py

```diff
@@ -12,7 +12,23 @@
 
 @implementer(IPublishTraverse)
 class GroupsPatch(Service):
-    """Updates an existing group."""
+    """Update an existing group with users, roles, groups, title and description.
```

Comment:
@ksuess nice one. I like the docstring. It would be extra nice to add this to our docs as well. :)

Not a blocker for a merge of course.

Reply:
Documentation is updated with common data to update on a group. 
https://github.com/plone/plone.restapi/pull/1439/commits/d30e9a132784373e1dd3dfaae27b28bf117dcdd2